mintapi/api.py

```diff
@@ -15,6 +15,7 @@
 import email
 import email.header
 import sys  # DEBUG
+import oathtool
```

Comment:
Can we do this import inside of the `if mfa_method == 'soft-token':` so that someone pulling the new code doesn't get broken without installing it?